js/views/modals/listingDetail/Listing.js

```diff
@@ -508,6 +549,11 @@ export default class extends BaseModal {
       this.setSelectedPhoto(this.activePhotoIndex);
       this.setActivePhotoThumbnail(this.activePhotoIndex);
       this.adjustPriceBySku();
+      this.$reviews = this.$('.js-reviews');
+
+      // get the ratings data, if any
```

Comment:
Also, there's no error handling here. For example, if a user has no reviews, the endpoint will error with:

```
{
    "success": false,
    "reason": "no link named 'index.json' under QmUNLLsPACCz1vLxQVkXqqLX5R1X345qqfHbsf67hvA3Nn"
}
```

In the case of an error, the first argument to always is the xhr which you end up passing to onRatings() which subsequently bombs with:

![image](https://user-images.githubusercontent.com/794517/27562230-786fdf42-5a80-11e7-9163-48cabeb8b29e.png)

I would think instead of always(), you would want to use done() and fail() so you could identify errors (not sure if we want to display them to the user or just not show reviews in that case - the latter may be fine for now, but ideally we handle it in a way that doesn't leave exceptions in the code.)